app/controllers/client_api/permissions_controller.rb

```diff
@@ -2,7 +2,7 @@ module ClientApi
   class PermissionsController < ApplicationController
     before_action :generate_permissions_object, only: :state
```

Comment:
Probably also need to change `:state` into `:status` here, right?

And also fix it if it needs to be fixed somewhere else, not sure (`routes.rb`, any front-end call?).